tests/robot-tests/tests/admin_and_public/bau/subject_reordering.robot

```diff
@@ -0,0 +1,246 @@
+    user checks accordion is in position    Four    4    id:uploadedDataFiles
+
+Add data guidance for all subjects
+    user clicks link    Data guidance
+    user waits until h2 is visible    Public data guidance
+
+    user waits until page contains element    id:dataGuidance-dataFiles
+    user waits until page contains accordion section    One
+
+    user waits until page contains element    id:dataGuidance-dataFiles
+    user checks accordion is in position    One    1    id:dataGuidance-dataFiles
+    user checks accordion is in position    Two    2    id:dataGuidance-dataFiles
+    user checks accordion is in position    Three    3    id:dataGuidance-dataFiles
+    user checks accordion is in position    Four    4    id:dataGuidance-dataFiles
+
+    user enters text into data guidance data file content editor    One
+    ...    One guidance content
+    user enters text into data guidance data file content editor    Two
+    ...    Two guidance content
+    user enters text into data guidance data file content editor    Three
+    ...    Three guidance content
+    user enters text into data guidance data file content editor    Four
+    ...    Four guidance content
+
+    user clicks button    Save guidance
+
+Publish release
+    user approves original release for immediate publication
+
+Check subject order in data tables
+    user navigates to data tables page on public frontend
+
+    user clicks radio    %{TEST_THEME_NAME}
+    user clicks radio    ${PUBLICATION_NAME}
+    user clicks element    id:publicationForm-submit
+
+    user waits until table tool wizard step is available    2    Choose a subject
+    user checks previous table tool step contains    1    Publication    ${PUBLICATION_NAME}
+
+    user checks page contains element
+    ...    xpath://*[contains(@class, "govuk-radios__item")][1]//label[contains(text(), "One")]
+    user checks page contains element
+    ...    xpath://*[contains(@class, "govuk-radios__item")][2]//label[contains(text(), "Two")]
+    user checks page contains element
+    ...    xpath://*[contains(@class, "govuk-radios__item")][3]//label[contains(text(), "Three")]
+    user checks page contains element
+    ...    xpath://*[contains(@class, "govuk-radios__item")][4]//label[contains(text(), "Four")]
+
+Check subject order in data catalogue
+    user navigates to data catalogue page on public frontend
+
+    user clicks radio    %{TEST_THEME_NAME}
+    user clicks radio    ${PUBLICATION_NAME}
+    user clicks button    Next step
+
+    user waits until page contains    Choose a release
+    user waits until page contains    ${RELEASE_NAME}
+
+    user clicks radio    ${RELEASE_NAME}
+    user clicks button    Next step
+
+    user waits until page contains    Choose files to download
+
+    user checks page contains element
+    ...    xpath://*[contains(@class, "govuk-checkboxes__item")][1]//label[contains(text(), "One")]
```

Comment:
Similar to above - would extract a keyword like `user checks checkbox exists` and use a better selector that doesn't rely on a class i.e `xpath://label[text()="${label}"]/../input[@type="radio"]`

Reply:
Did practically the same as with the radio case above, except there was no `data-testid` on check box items, so I added one.